io.itch.tx00100xt.SeriousSamClassic-VK.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+
+  <description>
+    <p>SeriousSamClassic-VK  is open source game engine version developed by Croteam for Serious Sam Classic  with  Vulkan support(Windows, Linux, FreeBSD, Raspberry Pi OS, macOS). Based on https://github.com/sultim-t/Serious-Engine-Vk and linux port https://github.com/icculus/Serious-Engine, but it should work on (or be easily portable to) any system that supports Vulkan and SDL2.
+    </p>
+    <p>Note that while the Serious-Engine source code has been released under GPL, you still need to legally own the game and provide Serious Sam Classic The First Encounter and Serious Sam Classic The Second Encounter the game data to play. See the How to Install section for more information.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_1.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_2.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_3.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_4.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_5.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samvulkan_6.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samxplus_1.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samxplus_2.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/tx00100xt/SeriousSamClassic-VK/raw/main/Images/samxplus_3.png</image>
+      <caption>SeriousSamClassic-VK running on Linux with XPLUS Mod</caption>
+    </screenshot>
+  </screenshots>
+
+  <releases>
+    <release version="1.10.6e" date="2024-01-25"/>
+    <release version="1.10.6d" date="2024-01-17"/>
+  </releases>
+
```

Comment:
also this fail to validate:

```
        "tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]"
```

Reply:
Latest release tagged 1.10.6d
1.10.6e tag only.  I'll fix it now